### ISSUE TITLE
EPIC - CLI : ps-upgrade:upgrade - implement post installation

### ISSUE THREAD

**Issue by fabienvallon** [PrestaShop/PrestaShop#35168]
As a user I want to upgrade prestashop so I can have an up2date software  

This is part of This is part of [CLI : ps-upgrade:upgrade](https://github.com/orgs/PrestaShop/projects/7/views/43?layout=board&pane=issue&itemId=44108684) 

## Description 

Verify everything is ok 

1. Disable maintenance
2. Diagnostic

### Disable maintenance 

See [maintenance](https://github.com/orgs/PrestaShop/projects/7/views/43?layout=board&pane=issue&itemId=44044496)

### Diagnostic 

See doctor  ? 

### Technical specs

- [ ] CliApp: Create this step in `ps-upgrade:shop:upgrade` command.

**Comment by Hlavtox** [comment]
Ping @Quetzacoalt91 Thomas, can I close this?

**Comment by Quetzacoalt91** [comment]
Yep, this has been replaced by Update Assistant 7 and the new step post update.
